src/nag-pack.ts

```diff
@@ -327,6 +339,7 @@ export abstract class NagPack implements IAspect {
     }
     line.push(params.level);
     line.push(params.info);
+    line.push(params.metadata ?? 'N/A');
```

Comment:
It seems like in many cases the report will have a `metadata` field which is full of `N/A`. I don't know if that is the best experience for users.